Bobfile

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
I had forgotten about this tool :smile:

Reply:
Heh, I haven't actually updated it in years, but every time I go back to anything docker, I reach straight for docker-builder.